Service/OS/AndroidGCMNotification.php

```diff
@@ -77,13 +85,15 @@ public function __construct($apiKey, $useMultiCurl, $timeout, $logger, AbstractC
     {
         $this->useDryRun = $dryRun;
         $this->apiKey = $apiKey;
+        // We'll need to set this per-request if `$client` was provided in constructor.
+        $this->clientOptions = array(
+            'timeout' => $timeout,
+            'verify_peer' => false,
```

Comment:
this shall be `verify` if you check https://github.com/kriswallsmith/Buzz/blob/master/lib/Client/AbstractClient.php#L91

Reply:
Good spot @jhkchan. I see [here](https://github.com/kriswallsmith/Buzz/blob/master/lib/Client/AbstractCurl.php#L200-L201) this option is actually also being used to toggle `CURLOPT_SSL_VERIFYHOST` too. This means the result is actually a further weakening of security checks vs. the old `setVerifyPeer(false)` call.

But since the new Buzz doesn't let you separate the two, and I am assuming there was a reason for the old option call, changing to the new option seems like the only safe fix for now.